MethodSwitch/src/methodswitch/MethodSwitch.java

```diff
@@ -0,0 +1,174 @@
+
+package methodswitch;
+
+import java.util.Scanner;
+
+/**
+ *
+ * @author Seth Frosch
+ */
+public class MethodSwitch {
+
+    public static void main(String[] args) {
+        int number;
+        Scanner keyboard = new Scanner(System.in);
+        
+        // Reformatted the menu options to give one per line.
+        System.out.println("Enter '1' to play FIZZBUZZ.\n" + 
+                "Enter '2' to view the value of your chosen word.\n" + 
+                "Enter '3' to use the salary calculator.\n" + 
+                "Enter '4' to check your grade.\n" + 
+                "Enter '5' to play Truth or Dare.\n" + 
+                "Enter '6' or any other number to EXIT.");
+        number = keyboard.nextInt();
+        
+        /**
+         * The switch method takes the user input number from 1 to 5 to run
+         * the appropriate method. 6 or any other number will end the program.
+         */
+        switch (number)
+        {
+            case 1:
+                fizzBuzz();
+                break;
+            case 2:
+                wordValue();
+                break;
+            case 3:
+                salaryCalculator();
+                break;
+            case 4:
+                checkGrade();
+                break;
+            case 5:
+                truthDare();
+                break;
+            default:
+                goodbye();
+        }
+    }
+    
+    /**
+     * Takes a user inputted number which if divisible by 3 outputs fizz,
+     * by 5 buzz, and both fizzbuzz. Neither gives an invalid statement.
+     */
+    public static void fizzBuzz(){
+        int fbNumber;
+        Scanner keyboard = new Scanner(System.in);
```

Comment:
Albeit repetitive I do like how you have a new instance of scanner in each function. The `new` operator is cheap and its ok to use it over and over again.